src/many-abci/src/abci_app.rs

```diff
@@ -68,12 +89,39 @@ impl AbciApp {
             debug!("Final migrations: {:?}", migrations);
             migrations
         });
+        let (transmitter, receiver) = channel::<transaction_cache::Message>();
+        std::thread::spawn(move || {
```

Comment:
Let me make sure I understand... So `transaction_cache` is a thread of the ABCI app that's spawned on launch. It's basically a hash map of transactions where we record key/values. It's just in memory and not persisted anywhere.

Reply:
That's correct.